cfe_internal/update/update_policy.cf

```diff
@@ -80,6 +80,57 @@ bundle agent cfe_internal_update_policy
 
 }
 
+bundle agent cfe_internal_remove_broken_symlink(symlink_path)
```

Comment:
Can't we remove this bundle now that your using move_obstructions?

Reply:
Yup, good catch.